src/containers/Bookings.js

```diff
@@ -4,33 +4,79 @@ import Results from "../components/Results.js";
 import * as FakeBookings from "../data/fakeBookings.json";
 
 export default class Bookings extends Component {
-  constructor(props){
+  constructor(props) {
     super(props);
-      this.state= {results:[]};
-
+    this.state = { results: [], direction: 'asc', style: {} };
 
+    this.addNumberOfDays(FakeBookings);
   }
+
+
+  addNumberOfDays = (FakeBookings) => {
+    FakeBookings.map((item) => {
```

Comment:
I think this would work, but it's not quite *semantic*. If we want to loop through all the list items and add something to them (like we are here), then I would use a `.forEach`. We use `.map` if we're creating a **new** array that is based on the old array, but we're transforming all of the items in the list.